admin.md

```diff
@@ -157,61 +206,85 @@ Both `tenantId` and `streamdId` **may only contain lowercase alpha numeric chara
-  1. `ingest.tenant24.data-int-1.qualifier1.in`
-  2. `ingest.tenant24.data-int-1.qualifier1.notification`
-  
-Note that the `numPartitions` and `retentionMs` topic configurations are required. There are other optional configurations that can also be passed in, see the [Stream Api Spec](apispec.md) for more details on these optional fields.
+This creates the following topics for you, both with one partition and a retention time of one day:
+  `ingest.tenant24.data-int-1.qualifier1.in`
+  `ingest.tenant24.data-int-1.qualifier1.notification`
+
+**Note:** The `numPartitions` and `retentionMs` topic configurations are required. You can pass other optional configurations. For information about these optional fields, see the Health Record Ingestion [API specification](apispec.md).
+
+#### Get Streams endpoint
 
-#### Get Streams
-The `Get` Streams endpoint takes in `tenantId` as a path parameter, and returns a list of all streamIds associated with that tenant. Assuming the above Create was run, then the following the following cURL command (HTTP/Get operation) would return a list containing the single streamId `data-int-1.qualifier1`:
+The `Get` Streams endpoint accepts `tenantId` as a path parameter, and returns a list of all streamIds associated with that tenant. Assuming the above Create was run, then this Curl command (HTTP/Get operation) returns a list containing the single streamId `data-int-1.qualifier1`:
 
       $ curl -X GET \
           <hri_base_url>/tenants/tenant24/streams \
           -H 'Accept: application/json' \
           -H 'Authorization: Bearer <token>' \
           -H 'X-IBM-Client-Id: <apikey>' \
           -H 'Content-Type: application/json'
-             
-#### Delete Stream
+
+#### Delete Stream endpoint
+
 Like `Create`, the `Delete` Stream endpoint takes in two path parameters, `tenantId` and `streamId`. The following curl command will delete both the `ingest.tenant24.data-int-1.qualifier1.in` and `ingest.tenant24.data-int-1.qualifier1.notification` topics:
 
       $ curl -X DELETE \
           <hri_base_url>/tenants/tenant24/streams/data-int-1.qualifier1 \
           -H 'Accept: application/json' \
-          -H 'Authorization: Bearer <token>' \
-          -H 'X-IBM-Client-Id: <apikey>' \
+          -H 'Authorization: Bearer <_token_>' \
+          -H 'X-IBM-Client-Id: <_apikey_>' \
           -H 'Content-Type: application/json' 
 
-Note that HRI topic naming conventions require topics to start with the prefix "ingest" and end with the suffix "in" or "notification". **Both the Get and Delete endpoints will ignore any topics that don't follow this convention**.
+**Important:** Topic naming conventions for Health Record Ingestion require topics to start with the prefix **ingest** and end with either suffix: **in** or **notification**. **Both the Get and Delete endpoints ignore any topics that do not follow this convention**.
```

Comment:
Change '**ingest**', '**in**' and '**notification**' to use ticks (`) instead of bold.